velox/docs/configs.rst

```diff
@@ -164,4 +167,35 @@ data size exceeds this limit, otherwise it spills the partition with most data.
 If the limit is zero, then the spiller always spills a previously spilled
 partition if it has any data. This is to avoid spill from a partition with a
 small amount of data which might result in generating too many small spilled
-files.
\ No newline at end of file
+files.
+
+
+Hive Configuration Properties
```

Comment:
Hive Connector (drop Configuration Properties as this is redundant).

Reply:
I still think this section should be named "Hive Connector".

Reply:
Will do